sdk/rag/document.go

```diff
@@ -37,7 +37,7 @@ func (c *Client) UploadDocumentsAndParse(ctx context.Context, datasetID string,
 }
 
 // UploadDocuments 上传文档（支持多文件和权限设置）
-func (c *Client) UploadDocuments(ctx context.Context, datasetID string, filePaths []string, groupIDs []int) ([]Document, error) {
+func (c *Client) UploadDocuments(ctx context.Context, datasetID string, filePaths []string, groupIDs []int, metadata *DocumentMetadata) ([]Document, error) {
 	var b bytes.Buffer
 	w := multipart.NewWriter(&b)
```

Comment:
建议在 `DocumentMetadata` 结构体中添加字段的注释，说明每个字段的用途和预期值范围。

```suggestion
type DocumentMetadata struct {
    // Title 文档标题
    Title string `json:"title"`
    // Author 文档作者
    Author string `json:"author"`
    // Description 文档描述
    Description string `json:"description"`
}